.github/CODEOWNERS

```diff
@@ -1,3 +1,25 @@
+# llvm
+/llvm/ @llvm/pr-subscribers-llvm
+
+# lld
+/lld/ @llvm/pr-subscribers-lld
+
+# compiler-rt
+/compiler-rt/ @llvm/pr-subscribers-compiler-rt
+
+# bolt
+/bolt/ @llvm/pr-subscribers-bolt
+
+# flang
+/flang/ @llvm/pr-subscribers-flang
+
+# llvm-libgcc
+/llvm-libgcc/ @llvm/pr-subscribers-llvm-libgcc
+
+# clang
+/clang/ @llvm/pr-subscribers-clang
+/clang-tools-extra/ @llvm/pr-subscribers-clang
```

Comment:
Note: I think pr-subscribers-clang should not watch /clang-tools-extra/. We can have a separate team to watch /clang-tools-extra